http/dotnet/httpclient/src/Middleware/Options/ChaosHandlerOption.cs

```diff
@@ -0,0 +1,30 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Microsoft.Kiota.Abstractions;
+
+namespace Microsoft.Kiota.Http.HttpClient.Middleware.Options
+{
+    /// <summary>
+    /// The Chaos Handler Option middleware class
+    /// </summary>
+    public class ChaosHandlerOption : IMiddlewareOption
+    {
+        /// <summary>
+        /// Percentage of responses that will have KnownChaos responses injected, assuming no PlannedChaosFactory is provided
+        /// </summary>
+        public int ChaosPercentLevel { get; set; } = 10;
+        /// <summary>
+        /// List of failure responses that potentially could be returned when 
+        /// </summary>
+        public List<HttpResponseMessage> KnownChaos { get; set; } = null;
```

Comment:
no need to default to null

Reply:
REsolved via https://github.com/microsoft/kiota/pull/609/commits/cf2ade9ca551518cdfc168fea220aae77cffb490